foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -95,6 +97,22 @@ public static OpenAiClient withCustomDestination(@Nonnull final Destination dest
     return new OpenAiClient(destination);
   }
 
+  /**
+   * Generate a completion for the given prompt.
```

Comment:
```suggestion
   * Generate a completion for the given user prompt.
```

to indicate that we will send a user message, not a system message